From 022635df5f06041897a8e2afeb3d292f51032ab1 Mon Sep 17 00:00:00 2001
From: David Schollmeyer <david.a.schollmeyer@intel.com>
Date: Sun, 29 Mar 2015 19:42:21 -0700
Subject: [PATCH 018/441] iosf-sb Add Cougar Mountain PCI device ID

Add device ID of CGM PCI bridge for IOSF-SB driver support on CE2700.
---
 arch/x86/kernel/iosf_mbi.c | 2 ++
 1 file changed, 2 insertions(+)

--- a/arch/x86/kernel/iosf_mbi.c
+++ b/arch/x86/kernel/iosf_mbi.c
@@ -30,6 +30,7 @@
 #define PCI_DEVICE_ID_BAYTRAIL		0x0F00
 #define PCI_DEVICE_ID_BRASWELL		0x2280
 #define PCI_DEVICE_ID_QUARK_X1000	0x0958
+#define PCI_DEVICE_ID_COUGAR_MTN 	0x2B80
 
 static DEFINE_SPINLOCK(iosf_mbi_lock);
 
@@ -292,6 +293,7 @@ static const struct pci_device_id iosf_m
 	{ PCI_DEVICE(PCI_VENDOR_ID_INTEL, PCI_DEVICE_ID_BAYTRAIL) },
 	{ PCI_DEVICE(PCI_VENDOR_ID_INTEL, PCI_DEVICE_ID_BRASWELL) },
 	{ PCI_DEVICE(PCI_VENDOR_ID_INTEL, PCI_DEVICE_ID_QUARK_X1000) },
+	{ PCI_DEVICE(PCI_VENDOR_ID_INTEL, PCI_DEVICE_ID_COUGAR_MTN) },
 	{ 0, },
 };
 MODULE_DEVICE_TABLE(pci, iosf_mbi_pci_ids);
